lib/event_store/config/parser.ex

```diff
@@ -39,7 +39,7 @@ defmodule EventStore.Config.Parser do
   defp parse_url(""), do: []
 
   defp parse_url(url) do
-    info = url |> URI.decode() |> URI.parse()
+    info = URI.parse(url)
```

Comment:
The docs for [`URI.decode/1`](https://hexdocs.pm/elixir/1.13/URI.html#decode/1) give this example:

```ex
iex> URI.decode("https%3A%2F%2Felixir-lang.org")
"https://elixir-lang.org"
```

Was that the purpose of this invocation?
The eventstore docs and tests don't mention that an encoded `DATABASE_URL` is supported, though.

As a reference, [Ecto.Repo](https://hexdocs.pm/ecto/Ecto.Repo.html#module-urls) also does not seem to support that.